package.json

```diff
@@ -94,7 +94,7 @@
     "test:unit": "jest"
   },
   "dependencies": {
-    "3box": "^1.22.0",
+    "3box": "1.20.2",
```

Comment:
why this is downgrade ?

Reply:
As I understand, versions conflicts with other newest packages. This is the easyest way to fix that conflict. But it can be also a problem of my local environment. I will also task it with latest version of 3box on Travis right now

Reply:
@orenyodfat when I try to install newest version it even not allow me to finish instalation. Just only this version works properly. Do you know, how to update it correctly?

Reply:
not sure.. it works ok with 1.22.0 before.

Reply:
So, I think, that it is conflict of packages. `3box` and `@wdio` I guess, but not sure

Reply:
@orenyodfat on my local machine it doesn't even allow me to install that package. Do you know anything about this library? Why do we need newest version?

Reply:
@orenyodfat I see, that it should improve performance: https://github.com/3box/3box-js/releases . Looks like it require additional settings on the local machine. Perhapse, I have incorrect JDK version installed locally or something similar

Reply:
@orenyodfat added [issue](https://github.com/3box/3box-js/issues/932) to the project

Reply:
@orenyodfat fixed rom linux machine